sample_apps/simple_app/instructions.md

```diff
@@ -23,69 +23,68 @@ $CERTIFIER_PROTOTYPE is the top level directory for the Certifier repository.
 It is helpful to have a shell variable for it, e.g., :
 
 ```shell
-$ export CERTIFIER_PROTOTYPE=~/src/github.com/jlmucb/crypto/v2/certifier-framework-for-confidential-computing
+export CERTIFIER_PROTOTYPE=~/src/github.com/jlmucb/crypto/v2/certifier-framework-for-confidential-computing
```

Comment:
You can skip reviewing this file.

I basically had 2nd thoughts on adding `$ ` prefix to each command.

When there are multiple commands shown, it's much easier to mouse-over and cut-and-paste all 3-4 commands in one go w/o the leading `$ `. 

So the diffs in this file are mainly to take out this leading `$ `